lib/options.js

```diff
@@ -11,12 +11,11 @@ module.exports = function parseOptions (params) {
   else if (Array.isArray(params.plugins)) plugins = params.plugins
   else plugins = params.plugins
 
-  const options = {}
-
-  if (typeof params !== 'undefined') {
-    options.parser = params.parser
-    options.syntax = params.syntax
-    options.stringifier = params.stringifier
+  const options = {
+    parser: params.parser,
+    syntax: params.syntax,
+    stringifier: params.stringifier,
+    map: params.sourceMap
```

Comment:
`postcss-load-config` return `map`, we should also return `map` from `parseOptions` (consistently and don't misleading for future options).